catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -54,10 +53,8 @@ public EnumerationValidator(final Set<String> values, final boolean ignoreCase)
 
         this.ignoreCase = ignoreCase;
 
-        this.values = Collections.unmodifiableSet(values.stream()
-                .filter(Objects::nonNull)
-                .map(ignoreCase ? String::toLowerCase : String::toString)
-                .collect(Collectors.toSet()));
+        this.values = new TreeSet<>(ignoreCase? String.CASE_INSENSITIVE_ORDER : null);
+        this.values.addAll(values);
```

Comment:
Could also change to something like...
```java
Supplier<Set<String>> supl =
        () -> ignoreCase ? new TreeSet<>(String.CASE_INSENSITIVE_ORDER) : new TreeSet();
this.values = values.stream()
        .filter(Objects::nonNull)
        .collect(Collectors.toCollection(supl));
```
...or take that and wrap in an unmodifiable collection is previously.